### DISCUSSION TITLE
Keda deployment without metrics server

### DISCUSSION THREAD

**Discussion by humutkazan** [kedacore/keda#5064] [Q&A / Need Help] [answered]
Hi all,

I just wonder if it is possible to use KEDA operator without metrics server. As far as I understand, metrics server is needed for HPA and HPA is required for 1 to N scaling. I don't need it. I just need 0 to 1 and vice versa activation. I tried disabling metrics server and saw that activation still worked but wanted to ask if that is valid practice and valid for production use.

Thanks in advance

**Comment by zroubalik** [comment, marked as answer]
If you need only activation, KEDA operator should be enough.
